apps.json

```
@@ -1506,7 +1510,7 @@
             "arbitrary-limitations"
         ],
         "category": "publishing",
-        "level": 8,
+        "level": 1,
```

```suggestion
        "level": 8,
```

as per https://ci-apps.yunohost.org/ci/job/12598